### ISSUE TITLE
Release Object-level checkpoints

### ISSUE THREAD

**Issue by hj-jung-01** [Haian-Jin/LVSM#2]
Hi, thank you for the nice work!

Could you please release object-level checkpoints(trained on Objaverse dataset) used in the paper?

Thank you.

**Comment by Haian-Jin** [comment]
Hi, since the object-level checkpoints were trained using internal Objaverse rendering data, I’m unable to release them at this time.

**Comment by hj-jung-01** [comment]
Could you give us tips for training on object-level dataset? Simply modifying dataloader seems not reproducing the paper results well.

**Comment by Haian-Jin** [comment]
You just need to modify the dataloader and adjust a few hyperparameters, which have been discussed in the paper. Please let me know what problems you are facing.

**Comment by hj-jung-01** [comment]
I think the problem may be caused by: (1) random sampling of 4 input views from the scene, and (2) handling RGBA images by simply slicing the first three RGB channels — should I be masking using the alpha channel instead?

**Comment by Haian-Jin** [comment]
- We also use random sampling for objects, which is also a common practice in object-level model training. You just need to random sample the input views in 360-degree 3D space. You can refer to many prior works to see how that works.

- Alpha blending is a standard operation when dealing with RGBA images. You should definitely apply alpha blending rather than simply discarding the alpha channel and using only the first three RGB channels.

**Comment by Haian-Jin** [comment]
yes, white background will be fine.

**Comment by hj-jung-01** [comment]
I think it works well now. Thank you :)

**Comment by jjordanoc** [comment]
Hi, @hj-jung-01 do you mind sharing your checkpoint on the Objaverse dataset? 
It would be really helpful 🙏🏻 
Thanks in advance :)
